devenv/src/command.rs

```diff
@@ -58,23 +58,21 @@ impl Devenv {
             }
             let error = cmd.exec();
             self.logger.error(&format!(
-                "Failed to replace shell with `{} {}`: {error}",
-                cmd.get_program().to_string_lossy(),
-                cmd.get_args()
-                    .map(|arg| arg.to_str().unwrap())
-                    .collect::<Vec<_>>()
-                    .join(" ")
+                "Failed to replace shell with `{}`: {error}",
+                display_command(&cmd),
             ));
             bail!("Failed to replace shell")
         } else {
-            let result = if options.logging {
+            if options.logging {
                 cmd.stdin(std::process::Stdio::inherit())
-                    .stderr(std::process::Stdio::inherit())
-                    .output()
-                    .expect("Failed to run command")
-            } else {
-                cmd.output().expect("Failed to run command")
-            };
+                    .stderr(std::process::Stdio::inherit());
```

Comment:
I'm not sure this works well with `devenv print-dev-env`?

Reply:
what do you mean? this part is functionally the same as before.